loggen/loggen.go

```diff
@@ -52,9 +52,10 @@ type LogGen struct {
 }
 
 type LogGenRequest struct {
-	Type   string `json:"type"`
-	Format string `json:"format"`
-	Count  int    `json:"count"`
+	Type    string `json:"type"`
+	Format  string `json:"format"`
+	Count   int    `json:"count"`
+	Framing bool   `json:"framing"`
```

Comment:
can we use omitempty here for compatibility?

Reply:
I was surprised too, but this is backward compatible. I don't know what the Gin framework does under the hood, but this bool defaults to false when missing from the JSON input.